forum/forum/routing.py

```diff
@@ -0,0 +1,22 @@
+from django.urls import path
+from channels.generic.websocket import AsyncWebsocketConsumer
+
+
+class Consumer(AsyncWebsocketConsumer):
+    async def connect(self):
+        await self.accept()
+        await self.send(text_data="Websocket connected.")
```

Comment:
Use `self.scope["user"]` if you're using `AuthMiddlewareStack`
This helps you validate authenticated users:
```
    async def connect(self):
        user = self.scope["user"]
        if user.is_authenticated:
            await self.accept()
            await self.send(text_data="WebSocket connected.")
        else:
            await self.close()
```
Makes the connection conditional, avoiding unauthorized access in future.